governance/communication-channels.md

```diff
@@ -0,0 +1,17 @@
+# Process for creating communication channels for TAG Environmental Sustainability
+
+This document describes the process for creating new communication channels for the CNCF TAG Environmental Sustainability (TAG ENV). Examples of such communication channels may be: Slack channels, accounts on social media platforms, etc.
+
+## Process for creating new Slack channels
+
+All channels must be created under [CNCF](https://slack.cncf.io) Slack. Currently TAG ENV has two Slack channels: main [#tag-environmental-sustainability](https://cloud-native.slack.com/archives/C03F270PDU6) channel and Green Reviews Working Group [#wg-green-reviews](https://cloud-native.slack.com/archives/C060EDHN431) channel.
+
+If a new Slack channel must be created, please ensure that following steps are followed as part of the process:
+
+* New channel is created under [CNCF](https://slack.cncf.io) Slack
+* At least one of the [TAG ENV Leads](https://github.com/cncf/tag-env-sustainability#tag-environmental-sustainability-co-chairs) has approved creation of the new channel
```

Comment:
I think we can open an issue for it. To discuss the purpose and reason. I dont think we will create new channels (slack and elsewhere) often so this should be fine.